core/domain/exp_services_test.py

```diff
@@ -7818,103 +7818,6 @@ def test_create_or_update_draft_with_exploration_model_not_created(
         self.assertEqual(exp_user_data.draft_change_list_exp_version, 1)
         self.assertEqual(exp_user_data.draft_change_list_id, 1)
 
-    def test_get_exp_with_draft_applied_when_draft_has_invalid_math_tags(
```

Comment:
Why is this test deleted?

Reply:
This test actually should return `None` when the math tag is invalid. This was failing as the invalid math tag was resulting in validation error and when I converted it to valid math tag then the test was not returning `None` as the tag is valid now. So @vojtechjelinek suggested that we can remove this particular test.
Thanks

Reply:
@lkbhitesh07 Did you tried swapping the validation function so that it didn't fail and we could properly test this?

Reply:
Yep! I forgot about that part. Edited now, the tests are passing.
Thanks